packages/charts/frontend/sirius-components-charts/src/bar/BarChart.types.ts

```diff
@@ -18,17 +18,13 @@ export interface BarChartProps {
 }
 
 export interface BarChartRepresentation {
-  metadata: RepresentationMetadata;
+  yAxisLabel: string;
```

Comment:
if the value is optional: the type should be `string | null`

Reply:
done